src/app/components/login/login.component.scss

```diff
@@ -3,10 +3,14 @@
     justify-content: center;
     align-items: center;
     height: 100%;
+    position: absolute;
```

Comment:
Not sure but i think absolute position is a bad practise, should ask Gytis on this one